db/migrate/20190524170042_create_rule_identifier_table.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRuleIdentifierTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rule_identifiers, id: :uuid do |t|
```

Comment:
Ditto

Reply:
I updated the RuleIdentifiers model for presence validations.